cpptests/t_client.cpp

```diff
@@ -90,23 +101,242 @@ class Client {
             destroyAndThrow();
         }
     }
+
     redisContext *ctx;
 };
 
-class ClientTest : public ::testing::Test {
+class ClientTestTCP : public ::testing::Test {
+public:
+    ClientTestTCP() : c(cs) {}
+
+protected:
+    virtual void SetUp() {        
+        c.flushdb();
+    }
+    virtual void TearDown() {
+    //    if(reply != nullptr) { freeReplyObject(reply); }
+        
+    }
+
+    ClientSettings cs;    
```

Comment:
This should be a global; if there needs to be overrides, it should be done on a per-function basis.

Reply:
Why?